src/plugins/hetero/executable_network.hpp

```diff
@@ -23,6 +23,15 @@
 #include "infer_request.hpp"
 #include "plugin.hpp"
 
+#include "src/plugin.hpp"
```

Comment:
Please fix code style and avoid `src/...`

Reply:
Done